docs/automating-github-org-management.md

```diff
@@ -0,0 +1,30 @@
+# Automating GitHub org management
+
+**TLDR; any changes that are applied manually using the GitHub UI will be reverted within one hour! Please file a PR against the [org configuration file]!**
+
+We are using [peribolos] to make transparent and automate management of our GitHub organization, covering members, org and repo settings, teams and access rights of teams to repositories.
+
+Our [org configuration file] contains the settings for our organization. These settings are synchronized by a periodic job.
+
+If you need to change settings within kubevirt org regarding
+
+- org settings, i.e.
+    - org admins
+    - org members
+- repos, i.e.
+    - new repos
+    - repo settings
+- teams, i.e.
+    - team maintainers and members
+    - team access to repositories
+
+please file a PR against the [org configuration file].
+
+After the PR got merged, those settings will get applied within one hour.
```

Comment:
Using `got` isn't good english.

Let's do...
`After the PR is merged changes will be applied within one hour.`